README.md

```diff
@@ -22,56 +21,55 @@ If the developer decides that a merge unit must not be merge the developer can i
+*  Create a cron job which periodically deletes logs older than X days in the log folder under `/var/subversion/mergetrigger/logs/`. E.g.: `find /var/subversion/mergetrigger/logs/ -type f -mtime +62 -delete`
 
 ## Client-Side
 
-### Prerequisites
-* Microsoft Windows (tested with 'Microsoft Windows 10 Professional x64')
-* AdoptOpenJDK 11 (tested with 'jdk-11.0.3+7')
-* Subversion 1.9.3 (tested with 'CollabNetSubversion Client 1.9.3 (r1718519)')
-* TortoiseSVN (tested with 'TortoiseSVN 1.9.3, Build 27038 - 64 Bit')
-
-### Installation
-* Place the content of the `client/bin/MergeProcessor_%VERSION%.7z` folder under the path `%ProgramFiles(x86)%\MergeProcessor`.
-* Create a shortcut in the start menu of user to the `mergeprocessor.exe`.
-* If the MergeProcessor should automatically start place a shortcut in the autostart folder of the user.
+### Prerequisites | Client-Side
+*  Microsoft Windows (tested with 'Microsoft Windows 10 Professional x64')
+*  AdoptOpenJDK 11 (tested with 'jdk-11.0.3+7')
+*  Subversion 1.9.3 (tested with 'CollabNetSubversion Client 1.9.3 (r1718519)')
+*  TortoiseSVN (tested with 'TortoiseSVN 1.9.3, Build 27038 - 64 Bit')
 
+### Installation | Client-Side
+*  Place the content of the `client/bin/MergeProcessor_%VERSION%.7z` folder under the path `%ProgramFiles(x86)%\MergeProcessor`.
+*  Create a shortcut in the start menu of user to the `mergeprocessor.exe`.
+*  If the MergeProcessor should automatically start place a shortcut in the autostart folder of the user.
 
 ### Configuration
-* Create a simple text file named `org.aposin.mergeprocessor.prefs` and save it under the path `%USERPROFILE%\AppData\Roaming\MergeProcessor`.
-* Define the following variables in a simple property format, e.g. `WORKING_FOLDER=C:\\dev\\mp\\`
-  * `WORKING_FOLDER` the folder where the MergeProcessor can create a working copy and store files.
-  * `SFTP_USERNAME` the username for accessing the SVN server (SFTP access, i.e. not used for SVN access)
-  * `SFTP_PASSWORD` the password for accessing the SVN server (SFTP access, i.e. not used for SVN access)
-  * `SFTP_HOST` the server where the merge units are stored.
-  * `SFTP_MERGEFOLDER` directory on the SVN server where to look for the merge units
-  * `GIT_REPOSITORIES_FOLDER` local directory where to clone required GIT repositories to merge on
-  * `RENAME_DATABASE_URL` JDBC URL to the renaming database (optional)
-  * `RENAME_DATABASE_USER` username for accessing the renaming database (optional)
-  * `RENAME_DATABASE_PASSWORD` password for accessing the renaming database (optional)
-  * `VERSION_INFO_FILES` files in the SVN repository for identifying the branch version (required when using renaming database). Multiple files can be defined, separated by a `,`.
+*  Create a simple text file named `org.aposin.mergeprocessor.prefs` and save it under the path `%USERPROFILE%\AppData\Roaming\MergeProcessor`.
```

Comment:
As codacy does not support nested lists correctly, let's change it to a simple list. I think we can merge the first 2 bullets together to a simple paragraph. Then we only have a simple list (no nesting) and codacy should hopefully be happy.

Would you be so kind to fix this? Then the README.md should be clean. 👍

Reply:
Hi Stefan. I've committed the changes. Let me know what you think 😄

Reply:
It's a bit different to what I expected but also a nice solution. 👍